packages/slice-machine/src/modules/update/index.ts

```diff
@@ -0,0 +1,36 @@
+import { Reducer } from "redux";
```

Comment:
I don't think we need a state for that : 
- "current" property is already present in the environment modules
- "recent" property is already present in the environment modules
- "update command" doesn't need to be stored in a global store, as we can get if from the packageCommand, this command should be handle by the NewVersionCommand
- "packageManager" should be added to the environment variable.

Reply:
Also I like the fact that the backend add an information on is an update is needed or not. So maybe we should add this to the env object too